src/App.tsx

```diff
@@ -1,61 +1,155 @@
+import { useEffect, useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+
+  const [todoTitle, setTodoTitle] = useState('');
+  const [selectedUserId, setSelectedUserId] = useState(0);
+
+  const [hasUserError, setHasUserError] = useState(false);
+  const [hasTodoTitleError, setHasTodoTitleError] = useState(false);
+
+  const getTodosWithUsers = () => {
+    const todosWithUsersToSet = [...todosFromServer]
```

Comment:
why do u need to make a copy?

Reply:
remove as map is non-mutable Array.prototype method